### DISCUSSION TITLE
About 'pytest-ordering' -> the value of 'order'

### DISCUSSION THREAD

**Discussion by lucky33newman** [pytest-dev/pytest#10527] [Q&A] [answered]
Hello guys,

I am using pytest-ordering to queue my tests up.

But I am facing a awkward situation:

I have a test that I would always put it as the last test.

But I am keeping adding new tests into my test framework, so, it is like, I had 12345 tests yesterday, but todays it is like total 12346 tests, I first have to give my new 12345 test 'test_balabala.py' an order 12345, then I change the order of the 'LAST_test.py', into new value order=12346, it is not smart right?

I know I could just give it a value that is super big like order=999999, but it is not smart enough. XD

Is there any smart way to place my LAST_test.py just as a last test?

Thanks.

:)

**Comment by The-Compiler** [comment, marked as answer]
pytest-ordering is unmaintained - I suggest using [pytest-order](https://pypi.org/project/pytest-order/) instead (even the pytest-ordering README suggests that). According to the readme of pytest-order, it supports:

> ordering of tests both from the start and from the end (via negative index)

So I suppose with that, `@pytest.mark.order(-1)` is all that's needed.

**Reply by lucky33newman** [reply]
That is exactly the smart way I desired. Much thanks.

**Reply by ssbarnea** [reply]
I made a request to archive the unmaintained repo.

**Reply by mrbean-bremen** [reply]
Just to give credit where credit is due: the feature of ordering from the end via negative index was already implemented in the original `pytest-ordering`.
